src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -4323,9 +4330,9 @@ int MacroAssembler::ic_check_size() {
 int MacroAssembler::ic_check(int end_alignment) {
   IncompressibleRegion ir(this);
   Register receiver = j_rarg0;
-  Register data = t1;
+  Register data = t0;
```

Comment:
As `ic_check` is used by `MachUEPNode::emit`, I think you need to update IC data register usage in `MachUEPNode::format` as well.

Reply:
Yes, good, fixed the prints.